src/main/java/org/isf/accounting/dto/BillDTO.java

```diff
@@ -57,7 +61,7 @@
 	
 	@NotNull
 	@ApiModelProperty(notes = "Is bill belongs to a patient?", example="true", position = 5)
-	private boolean isPatient;
+	private boolean patientBill;
```

Comment:
Are we sure of this? The core model has both:
```
	@NotNull
	@Column(name="BLL_IS_PAT")
	private boolean isPatient;
	
	@ManyToOne
	@JoinColumn(name="BLL_ID_PAT")
	private Patient billPatient;
```

Reply:
There was a name conflict in the generated method name and this was a way to resolve it.  I'll need to take a closer look.

Reply:
@mwithi Upon closer examination if `isPatient` is left as it was these are the methods generated:
```
	public @NotNull boolean isPatient() {
		return this.isPatient;
	}

	public void setPatient(@NotNull boolean isPatient) {
		this.isPatient = isPatient;
	}
```

But the `PatientDTO patient` variable generates
````
	public void setPatient(PatientDTO patient) {
		this.patient = patient;
	}
````
If I use maven then the compile is clean but if I try in the IDE compiling the `BillController` file with the annotations it complains with:
```
java: incompatible types: boolean cannot be converted to org.isf.patient.dto.PatientDTO
```
It appears that it cannot resolve the fact that there are 2 method signatures for `setPatient`.  This is probably a bug in the IDE but leaving it that way makes development at best inconvenient. 

If I de-lombok the file (that is expand all the generated methods) inline and try compiling in the IDE then it works fine which seems to validate the issue with annotation processing in IntelliJ when only using the generated code from the annotations.

So I've pushed another change to make `patientBill` into the variable `isPatientBill`.  I guess will need to live with it.

Reply:
To avoid such situations I use to name the variables of **ACertainType** with the type's name but lowercase, so something like:
```
private PatientDTO patientDTO;
```

For booleans, taking into account what the IDEs normally do, it's to say adding to a given <property> the words "is" and "set" in this way:
```
getter -> is<Property>
setter -> set<Property>
```

I think **isPatient** should be renamed to something more self-explaining, like:
```
private boolean patientTrue;
...
getter -> isPatientTrue();
setter -> setPatientTrue(boolean patientTrue);
```

Otherwise the proposed changes already work fine for me 👍